sctp/src/stream/mod.rs

```diff
@@ -698,60 +763,64 @@ impl AsyncRead for PollStream {
 impl AsyncWrite for PollStream {
     fn poll_write(
         self: Pin<&mut Self>,
-        _cx: &mut Context<'_>,
+        cx: &mut Context<'_>,
         buf: &[u8],
     ) -> Poll<io::Result<usize>> {
         let bytes = Bytes::copy_from_slice(buf);
-        match self.stream.write(&bytes) {
-            Ok(n) => Poll::Ready(Ok(n)),
-            Err(e) => Poll::Ready(Err(e.into())),
-        }
+
+        // This future, when polled, either succeeds in writing to the stream or registers the waker with the ressource that would block this from succeeding
```

Comment:
wouldn't it be simpler to revert back to https://github.com/webrtc-rs/webrtc/pull/344?

Reply:
True, and done :D